test/controllers/income/estimatedPay/update/IncomeUpdateBonusControllerSpec.scala

```diff
@@ -64,6 +64,8 @@ class IncomeUpdateBonusControllerSpec
       .thenReturn(Future.successful(Right(employer.id)))
     when(journeyCacheService.mandatoryJourneyValue(Matchers.eq(UpdateIncome_NameKey))(any()))
       .thenReturn(Future.successful(Right(employer.name)))
+    when(journeyCacheService.currentValue(Matchers.eq(UpdateIncome_TaxablePayKey))(any()))
+      .thenReturn(Future.successful(Some("taxablepay")))
```

Comment:
Should we add a test for when this is None that the url is 
controllers.income.estimatedPay.update.routes.IncomeUpdatePayslipAmountController.payslipDeductionsPage().url

Reply:
Yup, will do now :)